### ISSUE TITLE
Shape variation exploration by libcmaes? 

### ISSUE THREAD

**Issue by chatyan** [beniz/libcmaes#208]
I have seen the libcmaes, which is a good contribution. I am wondering if this library can be applied for real application.
For example the task of modeling a 3D model, based on genetic algorithm to find the best "fit" one with repect to a certain requirement.

I am not I am in the right direction, because I am lack of knowledge of GA and cma-es. So could you kindly give me some advise on this topic.

**Comment by beniz** [comment]
Yes, maybe look at the main CMA-ES R&D page with tutorials, applications, etc. http://cma.gforge.inria.fr/
